### ISSUE TITLE
OpenShift token retrieval from Keycloak should happen only once during workspace startup/suspension

### ISSUE THREAD

**Issue by skabashnyuk** [eclipse-che/che#19724]
### Is your task related to a problem? Please describe.
During workspace startup we performing multiple k8s operations.
For each operation, we requesting a keycloak for the new OpenShift token.
Some delays might happen between operations: image pulling, etc.
As a result, we might face the issue when the keycloak token is no longer valid.

### Describe the solution you'd like
OpenShift token retrieval from Keycloak should happen only once during workspace startup/suspension

### Describe alternatives you've considered
n/a
### Additional context
https://issues.redhat.com/browse/CRW-1767

**Comment by skabashnyuk** [comment]
won't fix
